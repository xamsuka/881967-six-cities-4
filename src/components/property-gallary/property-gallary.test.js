import React from 'react';
import renderer from 'react-test-renderer';
import PropertyGallary from './property-gallary.jsx';

const photos = [`https://canadskaya-izba.ru/img/doma/karkas2.jpg`];

test(`<PropertyGallary /> render`, () => {
  const three = renderer
    .create(<PropertyGallary photos = {photos} />).toJSON();

  expect(three).toMatchSnapshot();
});
