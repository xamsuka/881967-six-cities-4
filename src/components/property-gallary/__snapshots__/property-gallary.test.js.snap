// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<PropertyGallary /> render 1`] = `
<div
  className="property__gallery-container container"
>
  <div
    className="property__gallery"
  >
    <div
      className="property__image-wrapper"
    >
      <img
        alt="Photo studio"
        className="property__image"
        src="https://canadskaya-izba.ru/img/doma/karkas2.jpg"
      />
    </div>
  </div>
</div>
`;
